test/coverage/datamodel/UUriSerializerTest.cpp

```diff
@@ -23,16 +23,167 @@ class TestFixture : public testing::Test {
 
 	// Run once per execution of the test application.
 	// Used for setup of all tests. Has access to this instance.
-	TestFixture() = default;
-	~TestFixture() = default;
+	TestUUriSerializaerFixture() = default;
+	~TestUUriSerializaerFixture() = default;
 
 	// Run once per execution of the test application.
 	// Used only for global setup outside of tests.
 	static void SetUpTestSuite() {}
 	static void TearDownTestSuite() {}
 };
 
-// TODO replace
-TEST_F(TestFixture, SomeTestName) {}
+std::shared_ptr<uprotocol::v1::UUri> buildValidTestURI(
+    const std::string& authority = "192.168.1.10") {
+	std::shared_ptr<uprotocol::v1::UUri> uri =
+	    std::make_shared<uprotocol::v1::UUri>();
+	uri->set_authority_name(authority);
+	uri->set_ue_id(0x1100);
+	uri->set_ue_version_major(0xFF);
+	uri->set_resource_id(0x8000);
+	return uri;
+}
 
-}  // namespace
+// Positive test case - test serialization of UUri to string
+TEST_F(TestUUriSerializaerFixture, SerializeUUriToString) {
+	const std::shared_ptr<uprotocol::v1::UUri> testUUri = buildValidTestURI();
+	const std::string expectedUUri = "//192.168.1.10/1100/FF/8000";
+	const std::string actualUUri =
+	    uprotocol::datamodel::serializer::uri::AsString::serialize(*testUUri);
```

Comment:
It would be reasonable to add a `using uprotocol::datamodel::serializer;` to this file to shorten these lines.

Reply:
Done